packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/funman/FunmanController.java

```diff
@@ -141,4 +143,32 @@ public ResponseEntity<Simulation> createValidationRequest(
 
 		return ResponseEntity.ok(newSimulation);
 	}
+
+	@PutMapping("/{task-id}")
```

Comment:
Would be more appropriate and consistent to have this route as a DELETE instead of a PUT.

```
@DeleteMapping("/{task-id}")
```